app/src/Menus/FilterSection.js

```diff
@@ -0,0 +1,128 @@
+import React from "react";
+
+import Check from "../images/icons/Check-Circle.png";
+
+import styles from "./styles.module.scss";
+
+function FilterSection({ filterMenu, setFoodItems, fullMenu }) {
+  const [filters, setFilters] = React.useState([]);
+  const [showRestrictions, setShowRestrictions] = React.useState(true);
+  const [showOptions, setShowOptions] = React.useState(false);
+
+  //get list of filter items that are checked
+  const handleCheckedItem = (e) => {
+    const checkedOrNot = e.target.checked;
+    const checkedFilter = e.target.value;
+
+    //if check and the list does not include filter
+    if (checkedOrNot && !filters.includes(checkedFilter)) {
```

Comment:
Good use-case for `Set` to avoid checking if it's already in the list. Or you could filter for distinct values.

```js
setFilters(new Set([...filters, checkedFilter])); // always distinct

Array.from(filters); // if you need it back as an array
```